### PR TITLE
Modify: SlackTechnologies.Slack version 4.39.90

### LINKED ISSUE #162931: [Package Issue]: SlackTechnologies.Slack
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Other

### Brief description of your issue

Slack has an arm version https://slack.com/downloads/instructions/windows?ddl=1&build=arm64 but the winget package only specifies an x86 version. https://github.com/microsoft/winget-pkgs/blob/master/manifests/s/SlackTechnologies/Slack/4.39.90/SlackTechnologies.Slack.installer.yaml

I am happy to contribute if I could get pointed in the right direction.

### Steps to reproduce

N/A

### Actual behavior

Installing x86

### Expected behavior

Arm version installed

### Environment

```raw
Windows Package Manager v1.8.1911
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.26100.863
System Architecture: Arm64
Package: Microsoft.DesktopAppInstaller v1.23.1911.0

Winget Directories
-----------------------------------------------------------------------------------------------------------------------
Logs                               %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\Diag…
User Settings                      %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\sett…
Portable Links Directory (User)    %LOCALAPPDATA%\Microsoft\WinGet\Links
Portable Links Directory (Machine) C:\Program Files\WinGet\Links
Portable Package Root (User)       %LOCALAPPDATA%\Microsoft\WinGet\Packages
Portable Package Root              C:\Program Files\WinGet\Packages
Portable Package Root (x86)        C:\Program Files (x86)\WinGet\Packages
Installer Downloads                %USERPROFILE%\Downloads

Links
---------------------------------------------------------------------------
Privacy Statement   https://aka.ms/winget-privacy
License Agreement   https://aka.ms/winget-license
Third Party Notices https://aka.ms/winget-3rdPartyNotice
Homepage            https://aka.ms/winget
Windows Store Terms https://www.microsoft.com/en-us/storedocs/terms-of-sale

Admin Setting                             State
--------------------------------------------------
LocalManifestFiles                        Disabled
BypassCertificatePinningForMicrosoftStore Disabled
InstallerHashOverride                     Disabled
LocalArchiveMalwareScanOverride           Disabled
ProxyCommandLineOptions                   Disabled
DefaultProxy                              Disabled
```

### Screenshots and Logs

_No response_

### PR DESCRIPTION
**Pull request by SpecterShell**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

* Resolves #162931

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/162958)

### CODE REVIEW

File: manifests/s/SlackTechnologies/Slack/4.39.90/SlackTechnologies.Slack.installer.yaml [hunk 1]
```diff
@@ -1,32 +1,40 @@
...
   InstallerSha256: F23343217BA334CEF0850F2E6231AE15857772177C0379368D1BF55B814B180A
-  ProductCode: '{89D65F28-EDB6-4975-B339-EEBF903CD8B9}'
+  InstallerSwitches:
+    InstallLocation: APPLICATIONROOTDIRECTORY="<INSTALLPATH>"
+  ProductCode: '{89D65F28-EDB6-4975-B339-EEBF903CD8B9}.msq'
   AppsAndFeaturesEntries:
   - InstallerType: exe
+- Platform:
+  - Windows.Desktop
+  MinimumOSVersion: 10.0.18362.0
```
Comment (Trenly) [inline manifests/s/SlackTechnologies/Slack/4.39.90/SlackTechnologies.Slack.installer.yaml:32]:
```suggestion
  MinimumOSVersion: 10.0.17763.0
```

File: manifests/s/SlackTechnologies/Slack/4.39.90/SlackTechnologies.Slack.locale.en-US.yaml [hunk 2]
```diff
@@ -8,22 +8,32 @@ Publisher: Slack Technologies Inc.
...
 Description: Slack is a collaboration hub for work, no matter what work you do. It's a place where conversations happen, decisions are made, and information is always at your fingertips. With Slack, your team is better connected.
 Moniker: slack
 Tags:
-- communication
-- remote
-- slack
-- teams
-- teamwork
-- work
+- bussiness
```
Comment (jay-babu) [inline manifests/s/SlackTechnologies/Slack/4.39.90/SlackTechnologies.Slack.locale.en-US.yaml:22]:
```suggestion
- business

```

**Comment by jay-babu** [comment]
Thank you for this. There was no way I could figure this out

**Comment by mdanish-kh** [comment]
[Policy] Changes-Requested

https://github.com/microsoft/winget-pkgs/pull/162958#discussion_r1677258621

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
- https://github.com/microsoft/winget-pkgs/issues/142854

**Pull request merged**